MediaPushPlugin/src/main/resources/media-push-publisher.js

```diff
@@ -1 +1 @@
ow new Error("Called startBroadcasting while recording is in progress.");let t="",i=1280,r=720;void 0!==e.token&&(t=e.token),void 0!==e.width&&e.width>0&&(i=e.width),void 0!==e.height&&e.height>0&&(r=e.height);const a=await navigator.mediaDevices.getDisplayMedia({video:{frameRate:{max:30}},audio:{channelCount:2,echoCancellation:!1,autoGainControl:!1,noiseSuppression:!1},preferCurrentTab:!0}),n=a.getVideoTracks()[0];console.log("video track settings: ",n.getSettings());const o=a.getAudioTracks()[0];console.log("audio track settings: ",o.getSettings());const s={width:{min:640,ideal:i},height:{min:360,ideal:r},advanced:[{width:i,height:r},{aspectRatio:i/r}],frameRate:{min:15,max:30},resizeMode:"crop-and-scale"};n.applyConstraints(s),k=new g({websocket_url:e.websocketURL,peerconnection_config:{iceServers:[{urls:"stun:stun1.l.google.com:19302"}]},bandwidth:4e3,localStream:a,callback:(i,r)=>{"initialized"==i?(console.log("WebRTC adaptor initialized"),k.publish(e.streamId,t,"","","","",e.driverIp,"")):"publish_started"==i?(console.log("mediapush_publish_started"),P=!0):"publish_finished"==i&&(P=!1),console.log(i)},callbackError:function(e,t){var i=JSON.stringify(e);i=void 0!==t?t:JSON.stringify(e),-1!=e.indexOf("WebSocketNotConnected")?i="WebSocket is disconnected.":-1!=e.indexOf("not_initialized_yet")?i="Server is getting initialized.":-1!=e.indexOf("data_store_not_available")?i="Data store is not available. It's likely that server is initialized or getting closed":-1!=e.indexOf("NotFoundError")?i="Camera or Mic are not found or not allowed in your device":-1!=e.indexOf("NotReadableError")||-1!=e.indexOf("TrackStartError")?i="Camera or Mic are already in use and they cannot be opened. Choose another video/audio source if you have on the page below ":-1!=e.indexOf("OverconstrainedError")||-1!=e.indexOf("ConstraintNotSatisfiedError")?i="There is no device found that fits your video and audio constraints. You may change video and audio constraints":-1!=e.indexOf("NotAllowedError")||-1!=e.indexOf("PermissionDeniedError")?i="You are not allowed to access camera and mic.":-1!=e.indexOf("TypeError")?i="Video/Audio is required":-1!=e.indexOf("getUserMediaIsNotAllowed")?i="You are not allowed to reach devices from an insecure origin, please enable ssl":-1!=e.indexOf("ScreenSharePermissionDenied")?(i="You are not allowed to access screen share",$(".video-source").first().prop("checked",!0)):i=-1!=e.indexOf("UnsecureContext")?"Please Install SSL(https). Camera and mic cannot be opened because of unsecure context. ":-1!=e.indexOf("no_stream_exist")?"There is no active live stream with this id to play":e,void 0!==t&&console.log("error callback: "+e+" message: "+i)}}),window.webRTCAdaptorMediaPush=k},window.stopBroadcasting=function(e){k.stop(e.streamId),k.closeWebSocket()},window.isConnected=function(e){var t=!1;if(P){var i=k.signallingState(e);if(null!=i&&"closed"!=i){var r=k.iceConnectionState(e);null!=r&&"failed"!=r&&"disconnected"!=r&&(t=!0)}}return t}})()})();
```

Comment:
I think we can remove these files from repo becuase it's created on build time automatically